chalice/cli/__init__.py

```diff
@@ -201,11 +218,17 @@ def deploy(ctx, autogen_policy, profile, stage):
 @click.option('--include-lambda-messages/--no-include-lambda-messages',
               default=False,
               help='Controls whether or not lambda log messages are included.')
+@click.option('--stage', default='dev')
```

Comment:
No, not right now.  I'm hesitant to allow people to change the default to something like "prod".  I'd prefer if you're deploying from a script you have to run `--stage prod`.